openfe/protocols/openmm_afe/base.py

```diff
@@ -148,37 +148,76 @@ def _get_alchemical_indices(omm_top: openmm.Topology,
 
         return atom_ids
 
-    @staticmethod
-    def _pre_minimize(system: openmm.System,
-                      positions: omm_unit.Quantity) -> npt.NDArray:
+    def _pre_equilibrate(
```

Comment:
I'm wondering if we could modeularize the "run a simulation" part of #516 so this could just be imported here? Looking at the TODO items, seems like I'll just end up copying what @hannahbaumann already wrote.

Reply:
Yeah if that just defined an `MDUnit` then either protocol could issue that in their DAGs